app/assets/stylesheets/components/_headline-with-actions.scss

```diff
@@ -1,12 +1,11 @@
 .headline-with-actions {
   display: flex;
```

Comment:
I know I added these lines. So it's my fault. But we like to order our properties within a declaration block alphabetically. So it'd be ...

```
align-items: ...;
display: ...;
margin-bottom: ...;
```